<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackControls</class>
 <widget class="QWidget" name="PlaybackControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>52</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="timeSliderFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSlider" name="timeSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Simulation Time</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="displayFormat">
         <string>yyyy/MM/dd HH:mm:ss.zzz</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="playFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="playSpeedLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="slowDownButton">
          <property name="text">
           <string>Slow Down</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="speedUpButton">
          <property name="text">
           <string>Speed Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="maxSpeedButton">
          <property name="text">
           <string>MAX</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="goToBeginningButton">
        <property name="toolTip">
         <string>Go To Beginning</string>
        </property>
        <property name="text">
         <string>Go To Beginning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="playPauseButton">
        <property name="toolTip">
         <string>Play</string>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="goToEndButton">
        <property name="toolTip">
         <string>Go To End</string>
        </property>
        <property name="text">
         <string>Go To End</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="speedSliderLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="slowerLabel">
          <property name="text">
           <string>Slower</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="speedSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Toggle Play Speed</string>
          </property>
          <property name="minimum">
           <number>-5</number>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBothSides</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fasterLabel">
          <property name="text">
           <string>Faster</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionGoToBeginning">
   <property name="text">
    <string>Go To Beginning</string>
   </property>
   <property name="toolTip">
    <string>Go To Beginning</string>
   </property>
  </action>
  <action name="actionPlayPause">
   <property name="text">
    <string>PlayPause</string>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionGoToEnd">
   <property name="text">
    <string>Go To End</string>
   </property>
   <property name="toolTip">
    <string>Go To End</string>
   </property>
  </action>
  <action name="actionSpeedUp">
   <property name="text">
    <string>Speed Up</string>
   </property>
   <property name="toolTip">
    <string>Speed Up (2x)</string>
   </property>
   <property name="shortcut">
    <string>.</string>
   </property>
  </action>
  <action name="actionSlowDown">
   <property name="text">
    <string>Slow Down</string>
   </property>
   <property name="toolTip">
    <string>Slow Down (2x)</string>
   </property>
   <property name="shortcut">
    <string>,</string>
   </property>
  </action>
  <action name="actionMaxSpeed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Max Speed</string>
   </property>
   <property name="toolTip">
    <string>Max Speed</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
